Zend/zend_closures.c

```diff
@@ -18,15 +18,12 @@
    +----------------------------------------------------------------------+
 */
 
-#include "zend.h"
-#include "zend_API.h"
 #include "zend_closures.h"
-#include "zend_exceptions.h"
-#include "zend_interfaces.h"
-#include "zend_objects.h"
-#include "zend_objects_API.h"
-#include "zend_globals.h"
-#include "zend_closures_arginfo.h"
+#include "zend_API.h" //for ZEND_METHOD()
+#include "zend_arena.h"
+#include "zend_compile.h" // for union _zend_function
+#include "zend_objects.h" // for zend_object_std_init()
+#include "zend_closures_arginfo.h" // for register_class_Closure()
```

Comment:
I documented only the first or most prominent use of that header, even if more symbols were used, because I thought that's enough (a trade-off between having something vs having a line too long with too much noisy information). But in this case, the comment is indeed expendable because it's obvious enough. Removing the comment.